F1WM/Services/CalendarService.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using F1WM.ApiModel;
+using F1WM.Repositories;
+
+namespace F1WM.Services
+{
+    public class CalendarService : ICalendarService
+    {
+        private readonly ICalendarRepository repository;
+        public async Task<Calendar> GetCalendar(int? year)
+        {
+            var news = await repository.GetCalendar(year.HasValue ? year.Value : DateTime.Now.Year);

```

Comment:
Please use ITimeService here to get the current date. It helps in testing (you don't have to rely on the time when test are being run).